Dockerfile

```diff
@@ -10,19 +12,38 @@ RUN apt-get update && apt-get install -y -q eatmydata \
         zlib1g-dev \
     && rm -rf /var/lib/apt/lists/*
 
+# Install newer cmake version with globally scoped targets
+ENV CMAKE_VER=3.13.3
```

Comment:
This should be 3.10.2 if that is the version that we are targeting for support.

Reply:
The fact that it was missing the export set stopped me from using it. Agreed though, I will change this to 3.10 instead.